golang/cosmos/x/vstorage/keeper/keeper.go

```diff
@@ -119,13 +119,19 @@ func NewKeeper(storeKey storetypes.StoreKey) Keeper {
 	}
 }
 
-var MetricKeysSizeDelta = []string{"store", "size_delta"}
+var MetricKeyStoreAllocated = []string{"store", "allocated"}
+var MetricKeyStoreReleased = []string{"store", "released"}
```

Comment:
These names will potentially be confusing out of context. Let's dumb them down:
```suggestion
var MetricKeyStoreSizeIncrease = []string{"store", "size_increase"}
var MetricKeyStoreSizeDecrease = []string{"store", "size_decrease"}
```

Reply:
done.